src/_components/lang.ejs

```diff
@@ -0,0 +1,6 @@
+<div class="c-lang">
```

Comment:
<img width="169" alt="Screenshot 2022-07-08 at 15 17 19" src="https://user-images.githubusercontent.com/7137128/177990601-480e3fc5-4ed2-48c7-b978-138ea0e4a119.png">
Isn't it a &lt;select&gt; element?